tools/webpack/experimental.config.js

```diff
@@ -0,0 +1,425 @@
+			},
+		} ),
+		new RtlCssPlugin( {
+			filename: ( pathData ) => {
+				if (
+					GUTENBERG_PACKAGES.find(
+						( packageName ) =>
+							pathData.chunk.name === `${ packageName }/index`
+					)
+				) {
+					return pathData.chunk.name.replace(
+						'/index',
+						'/style-rtl.css'
+					);
+				}
+				return '[name]-rtl.css';
+			},
+		} ),
+		new CopyWebpackPlugin( {
+			patterns: [].concat(
+				Object.entries( {
+					'./packages/block-library/src/': 'block-library/blocks/',
+					'./packages/edit-widgets/src/blocks/':
+						'edit-widgets/blocks/',
+					'./packages/widgets/src/blocks/': 'widgets/blocks/',
+				} ).flatMap( ( [ from, to ] ) => [
+					{
+						from: `${ from }/**/index.php`,
+						to( { absoluteFilename } ) {
+							const [ , dirname ] = absoluteFilename.match(
+								new RegExp(
+									`([\\w-]+)${ escapeRegExp(
+										sep
+									) }index\\.php$`
+								)
+							);
+
+							return join( to, `${ dirname }.php` );
+						},
+						transform: ( content ) => {
+							content = content.toString();
+
+							// Within content, search for any function definitions. For
+							// each, replace every other reference to it in the file.
+							return (
+								Array.from(
+									content.matchAll(
+										/^\s*function ([^\(]+)/gm
+									)
+								)
+									.reduce( ( result, [ , functionName ] ) => {
+										// Prepend the Gutenberg prefix, substituting any
+										// other core prefix (e.g. "wp_").
+										return result.replace(
+											new RegExp( functionName, 'g' ),
+											( match ) =>
+												'gutenberg_' +
+												match.replace( /^wp_/, '' )
+										);
+									}, content )
+									// The core blocks override procedure takes place in
+									// the init action default priority to ensure that core
+									// blocks would have been registered already. Since the
+									// blocks implementations occur at the default priority
+									// and due to WordPress hooks behavior not considering
+									// mutations to the same priority during another's
+									// callback, the Gutenberg build blocks are modified
+									// to occur at a later priority.
+									.replace(
+										/(add_action\(\s*'init',\s*'gutenberg_register_block_[^']+'(?!,))/,
+										'$1, 20'
+									)
+							);
+						},
+						noErrorOnMissing: true,
+					},
+					{
+						from: `${ from }/*/block.json`,
+						to( { absoluteFilename } ) {
+							const [ , dirname ] = absoluteFilename.match(
+								new RegExp(
+									`([\\w-]+)${ escapeRegExp(
+										sep
+									) }block\\.json$`
+								)
+							);
+
+							return join( to, dirname, 'block.json' );
+						},
+					},
+				] )
+			),
+		} ),
+		new ForkTsCheckerWebpackPlugin( {
+			typescript: {
+				diagnosticOptions: {
+					semantic: true,
+					syntactic: true,
+				},
+				mode: 'write-references',
+				build: true,
+			},
+		} ),
+		new IgnoreEmitPlugin( /\.css-entry\.js$/ ),
```

Comment:
We're ignoring the unused CSS entry js for now since they serve no purposes. However, in the future, we could use them for hot reloading CSS during development.